governance-server/src/delegation_logic.js

```diff
@@ -16,12 +15,12 @@ delegationLogic.findDelegationEvents = async (cache, aeternity, height, setCache
         const registryContractTransactions = await contractTransactionHashes();
         const registryContractEvents = await registryContractTransactions.asyncMap(aeternity.transactionEvent);
 
-        const sortedDelegationEvents = registryContractEvents
+        const filteredContractEvents = registryContractEvents
             .filter(event => event && event.height <= height)
             .filter(event => ["Delegation", "RevokeDelegation"].includes(event.topic))
-            .sort((a, b) => a.height - b.height || a.nonce - b.nonce);
 
-        return sortedDelegationEvents
+        filteredContractEvents.sort((a, b) => a.height - b.height || a.nonce - b.nonce);
```

Comment:
whats the reason for removing the inline sorting?

Reply:
sort mutates the array it is called upon, as far as I did see this is the default way of using it.

Reply:
it mutates the array but also returns it, which is why the previous implementation should work just fine

Reply:
yeah just changed it during my work to debug and didn't change it back, should I revert or leave as is?

Reply:
I would say revert, its a cleaner code